### ISSUE TITLE
chain.delete_rule fails with iptc.xtables.XTablesError: conntrack: no such parameter state

### ISSUE THREAD

**Issue by RickDavis84** [ldx/python-iptables#153]
I'm unable to delete a rule that has a tcp & conntrack match, but it works if the rules don't have the statematch added to them...this may be related to #46 

Linux Version:
```
# uname -a
Linux ubu-dt-1504 3.19.0-21-generic #21-Ubuntu SMP Sun Jun 14 18:31:11 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```
IPTables Version:
```
# iptables --version
iptables v1.4.21
```
IPTC - Failed with both 0.8.0 and 0.9.0-dev:
```
>>> print(iptc.version.__version__)
0.8.0
```
```
>>> print(iptc.version.__version__)
0.9.0-dev
```
Rough Source Code:
```
import sys
import socket
import os
import iptc

def deleteRulesByPort(protocol, port, mychain):
    table = iptc.Table(iptc.Table.FILTER)
    table.autocommit = False
    chain = iptc.Chain(table, 'INPUT')
    for myrule in chain.rules:
        matched = False
        if myrule.protocol == protocol:
            for match in myrule.matches:
                if match.dport == port or match.sport==port:
                    matched = True
        if matched:
            print("Deleting: ", myrule.src, myrule.dst)
            chain.delete_rule(myrule)
            print("deleted")
    table.commit()
    table.autocommit = True

def addRule(chain, target, protocol, src, dst, states, proto_src_port = '', proto_dst_port = ''):
    mychain = iptc.Chain(iptc.Table(iptc.Table.FILTER), chain)

    rule=iptc.Rule()
    rule.protocol = protocol
    rule.src = src
    rule.dst = dst
    rule.target = iptc.Target(rule, target)

    statematch = iptc.Match(rule, 'state')
    statematch.state = states
    rule.add_match(statematch)

    portmatch = iptc.Match(rule, protocol)

    if proto_src_port:
        portmatch.sport = proto_src_port
    elif proto_dst_port:
        portmatch.dport = proto_dst_port
    rule.add_match(portmatch)
    mychain.insert_rule(rule)

def main(argv=None): # IGNORE:C0111
    for host in socket.gethostbyname_ex('google.com')[2]:
        print("Adding host: " + host)
        addRule(chain='OUTPUT', target='ACCEPT', protocol='tcp', proto_dst_port='80', src='127.0.0.1', dst=host, states='NEW,ESTABLISHED')
        addRule(chain='INPUT', target='ACCEPT', protocol='tcp', proto_src_port='80', src=host, dst='127.0.0.1', states='ESTABLISHED,RELATED')
    deleteRulesByPort(protocol='tcp', port='80', mychain='INPUT')

    return 0
```
Error Message:
```
# python /home/rdavis/workspace/iptables-dnsupdater/iptables-dnsupdater/iptables-dnsupdater.py 
Adding host: 216.58.217.206
('Deleting: ', '216.58.217.206/255.255.255.255', '127.0.0.1/255.255.255.255')
Traceback (most recent call last):
  File "/home/rdavis/workspace/iptables-dnsupdater/iptables-dnsupdater/iptables-dnsupdater.py", line 235, in <module>
    sys.exit(main())
  File "/home/rdavis/workspace/iptables-dnsupdater/iptables-dnsupdater/iptables-dnsupdater.py", line 128, in main
    deleteRulesByPort(protocol='tcp', port='80', mychain='INPUT')
  File "/home/rdavis/workspace/iptables-dnsupdater/iptables-dnsupdater/iptables-dnsupdater.py", line 85, in deleteRulesByPort
    chain.delete_rule(myrule)
  File "/usr/local/lib/python2.7/dist-packages/iptc/ip4tc.py", line 1419, in delete_rule
    rule.final_check()
  File "/usr/local/lib/python2.7/dist-packages/iptc/ip4tc.py", line 928, in final_check
    match.final_check()
  File "/usr/local/lib/python2.7/dist-packages/iptc/ip4tc.py", line 335, in final_check
    self._update_parameters()
  File "/usr/local/lib/python2.7/dist-packages/iptc/ip4tc.py", line 439, in _update_parameters
    self.__setattr__(k, v)
  File "/usr/local/lib/python2.7/dist-packages/iptc/ip4tc.py", line 443, in __setattr__
    self.parse(name.replace("_", "-"), value)
  File "/usr/local/lib/python2.7/dist-packages/iptc/ip4tc.py", line 328, in parse
    self._parse(argv, inv, entry)
  File "/usr/local/lib/python2.7/dist-packages/iptc/ip4tc.py", line 581, in _parse
    self._orig_parse, self._orig_options)
  File "/usr/local/lib/python2.7/dist-packages/iptc/xtables.py", line 739, in new
    ret = fn(*args)
  File "/usr/local/lib/python2.7/dist-packages/iptc/xtables.py", line 1039, in parse_match
    argv[0]))
iptc.xtables.XTablesError: conntrack: no such parameter state
```
IPTables output:
```
# iptables -nL
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         
ACCEPT     tcp  --  216.58.217.206       127.0.0.1            state RELATED,ESTABLISHED tcp spt:80

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
ACCEPT     tcp  --  127.0.0.1            216.58.217.206       state NEW,ESTABLISHED tcp dpt:80
```
If you comment out the statematch lines in addRule, flush the iptables rules and re-run the script, the chain.delete_rule call will work.
```
def addRule(chain, target, protocol, src, dst, states, proto_src_port = '', proto_dst_port = ''):
...
...
#    statematch = iptc.Match(rule, 'state')
#    statematch.state = states
#    rule.add_match(statematch)
...
...
```

**Comment by ldx** [comment]
Can you give latest master a shot? Your example code works fine here with it.

**Comment by RickDavis84** [comment]
When I pulled the latest version it was 0.9.0-dev, which gave me the same error, did it work for you on ubuntu 15.04?

**Comment by ldx** [comment]
Can you check what commit master was at when you cloned? I tested this on debian 8, but I can also set up an ubuntu vm.

**Comment by RickDavis84** [comment]
I pulled the latest from this morning (about an hour before submitting this issue)...

# git rev-parse --verify HEAD
545069b570e4606e9d80a0d78c864defe75b957d

**Comment by RickDavis84** [comment]
I just tested on a CentOS 6.6 box with python2.6.6 and it worked fine with both iptc 0.8.0 and 0.9.0-dev, must be something specific with ubuntu

```
# uname -a
Linux bastcentos66-01 2.6.32-504.el6.x86_64 #1 SMP Wed Oct 15 04:27:16 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux

# python --version
Python 2.6.6
```

**Comment by RickDavis84** [comment]
I was also able to reproduce this bug on CentOS 7 which has iptables v1.4.21...have you been able to reproduce the issue?

```
[rdavis@localhost iptables-dnsupdater]$ cat /etc/redhat-release 
CentOS Linux release 7.1.1503 (Core) 

[rdavis@localhost iptables-dnsupdater]$ uname -a
Linux localhost.localdomain 3.10.0-229.7.2.el7.x86_64 #1 SMP Tue Jun 23 22:06:11 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

[rdavis@localhost iptables-dnsupdater]$ python --version
Python 2.7.5

[rdavis@localhost iptables-dnsupdater]$ iptables --version
iptables v1.4.21

[rdavis@localhost iptables-dnsupdater]$ python
Python 2.7.5 (default, Jun 24 2015, 00:41:19) 
[GCC 4.8.3 20140911 (Red Hat 4.8.3-9)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import iptc
>>> print iptc.version.__version__
0.8.0
```

**Comment by ldx** [comment]
Yes, I see the same.

**Comment by ldx** [comment]
@RickDavis84 branch `issue153` has a fix. Let me know if it fixes your problem.

**Comment by RickDavis84** [comment]
That fixed the issue on my ubuntu box, haven't tested the centos6.6 one yet but I'm sure it's fine. Thanks for the super fast bug fix!

**Comment by RickDavis84** [comment]
It looks like this works well on CentOS 7, which has iptables v1.4.21/Python 2.7.5, but CentOS 6.5 & CentOS 6.6 both get a glibc error if I add a rule which has both a TCP port match and a statematch. The rule gets added into iptables, but because of the glibc error my script crashes and I can't add more rules.

I tried using python 2.7.10 in a virtualenv on the centos6.5 & 6.6 boxes, but it didn't help. If you disable gc and then call gc.collect() it will throw the error after the rule add has been committed. Otherwise it will fail once your iptc.Table object goes out of scope & gc tries to reclaim it. 


```
[root@centos70 ~]# uname -a && iptables --version && python --version
Linux centos70 3.10.0-123.el7.x86_64 #1 SMP Mon Jun 30 12:09:22 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
iptables v1.4.21
Python 2.7.5

centos70 works well with the latest version of iptables



[root@centos65 ~]# uname -a && iptables --version && python --version
Linux centos65 2.6.32-431.el6.x86_64 #1 SMP Fri Nov 22 03:15:09 UTC 2013 x86_64 x86_64 x86_64 GNU/Linux
iptables v1.4.7
Python 2.6.6

[vagrant@centos66 ~]$ uname -a && iptables --version && python --version
Linux centos66 2.6.32-504.el6.x86_64 #1 SMP Wed Oct 15 04:27:16 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
iptables v1.4.7
Python 2.6.6


*** glibc detected *** python: free(): invalid pointer: 0x00007fe2ac4ee2a0 ***
======= Backtrace: =========
/lib64/libc.so.6(+0x75e66)[0x7fe2b799ce66]
/usr/lib64/libffi.so.5(ffi_call_unix64+0x4c)[0x7fe2ae014dac]
/usr/lib64/libffi.so.5(ffi_call+0x214)[0x7fe2ae014b34]
/usr/lib64/python2.6/lib-dynload/_ctypes.so(_CallProc+0x394)[0x7fe2ae228074]
/usr/lib64/python2.6/lib-dynload/_ctypes.so(+0xa3a2)[0x7fe2ae2213a2]
/usr/lib64/libpython2.6.so.1.0(PyObject_Call+0x53)[0x7fe2b85a6c63]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x47d4)[0x7fe2b8637f74]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x927)[0x7fe2b863a657]
/usr/lib64/libpython2.6.so.1.0(+0x6acb0)[0x7fe2b85cdcb0]
/usr/lib64/libpython2.6.so.1.0(PyObject_Call+0x53)[0x7fe2b85a6c63]
/usr/lib64/libpython2.6.so.1.0(+0x566af)[0x7fe2b85b96af]
/usr/lib64/libpython2.6.so.1.0(PyObject_Call+0x53)[0x7fe2b85a6c63]
/usr/lib64/libpython2.6.so.1.0(PyEval_CallObjectWithKeywords+0x43)[0x7fe2b8632c93]
/usr/lib64/libpython2.6.so.1.0(+0x95ffe)[0x7fe2b85f8ffe]
/usr/lib64/libpython2.6.so.1.0(+0x9a57f)[0x7fe2b85fd57f]
/usr/lib64/libpython2.6.so.1.0(+0x79e4b)[0x7fe2b85dce4b]
/usr/lib64/libpython2.6.so.1.0(+0x9a75c)[0x7fe2b85fd75c]
/usr/lib64/libpython2.6.so.1.0(+0x70aff)[0x7fe2b85d3aff]
/usr/lib64/libpython2.6.so.1.0(PyDict_Clear+0x10b)[0x7fe2b85de9cb]
/usr/lib64/libpython2.6.so.1.0(+0x7ba39)[0x7fe2b85dea39]
/usr/lib64/libpython2.6.so.1.0(+0x10071e)[0x7fe2b866371e]
/usr/lib64/libpython2.6.so.1.0(+0x100ec9)[0x7fe2b8663ec9]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x5244)[0x7fe2b86389e4]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x927)[0x7fe2b863a657]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x5304)[0x7fe2b8638aa4]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x927)[0x7fe2b863a657]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x5304)[0x7fe2b8638aa4]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x927)[0x7fe2b863a657]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalFrameEx+0x5304)[0x7fe2b8638aa4]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalCodeEx+0x927)[0x7fe2b863a657]
/usr/lib64/libpython2.6.so.1.0(PyEval_EvalCode+0x32)[0x7fe2b863a732]
/usr/lib64/libpython2.6.so.1.0(+0xf1bac)[0x7fe2b8654bac]
/usr/lib64/libpython2.6.so.1.0(PyRun_FileExFlags+0x90)[0x7fe2b8654c80]
/usr/lib64/libpython2.6.so.1.0(PyRun_SimpleFileExFlags+0xdc)[0x7fe2b865616c]
/usr/lib64/libpython2.6.so.1.0(Py_Main+0xb62)[0x7fe2b86628a2]
/lib64/libc.so.6(__libc_start_main+0xfd)[0x7fe2b7945d5d]
python[0x400649]
======= Memory map: ========
00400000-00401000 r-xp 00000000 fd:00 2361391                            /usr/bin/python
00600000-00601000 rw-p 00000000 fd:00 2361391                            /usr/bin/python
0132a000-01738000 rw-p 00000000 00:00 0                                  [heap]
7fe2abcd3000-7fe2abce9000 r-xp 00000000 fd:00 2228226                    /lib64/libgcc_s-4.4.7-20120601.so.1
7fe2abce9000-7fe2abee8000 ---p 00016000 fd:00 2228226                    /lib64/libgcc_s-4.4.7-20120601.so.1
7fe2abee8000-7fe2abee9000 rw-p 00015000 fd:00 2228226                    /lib64/libgcc_s-4.4.7-20120601.so.1
7fe2abee9000-7fe2abeea000 r-xp 00000000 fd:00 2228525                    /lib64/xtables/libxt_standard.so
7fe2abeea000-7fe2ac0e9000 ---p 00001000 fd:00 2228525                    /lib64/xtables/libxt_standard.so
7fe2ac0e9000-7fe2ac0ea000 rw-p 00000000 fd:00 2228525                    /lib64/xtables/libxt_standard.so
7fe2ac0ea000-7fe2ac0ec000 r-xp 00000000 fd:00 2228529                    /lib64/xtables/libxt_tcp.so
7fe2ac0ec000-7fe2ac2ec000 ---p 00002000 fd:00 2228529                    /lib64/xtables/libxt_tcp.so
7fe2ac2ec000-7fe2ac2ed000 rw-p 00002000 fd:00 2228529                    /lib64/xtables/libxt_tcp.so
7fe2ac2ed000-7fe2ac2ee000 r-xp 00000000 fd:00 2228526                    /lib64/xtables/libxt_state.so
7fe2ac2ee000-7fe2ac4ee000 ---p 00001000 fd:00 2228526                    /lib64/xtables/libxt_state.so
7fe2ac4ee000-7fe2ac4ef000 rw-p 00001000 fd:00 2228526                    /lib64/xtables/libxt_state.so
7fe2ac4ef000-7fe2ac4f4000 r-xp 00000000 fd:00 2228251                    /lib64/libnss_dns-2.12.so
7fe2ac4f4000-7fe2ac6f3000 ---p 00005000 fd:00 2228251                    /lib64/libnss_dns-2.12.so
7fe2ac6f3000-7fe2ac6f4000 r--p 00004000 fd:00 2228251                    /lib64/libnss_dns-2.12.so
7fe2ac6f4000-7fe2ac6f5000 rw-p 00005000 fd:00 2228251                    /lib64/libnss_dns-2.12.so
7fe2ac6f5000-7fe2ac701000 r-xp 00000000 fd:00 2228253                    /lib64/libnss_files-2.12.so
7fe2ac701000-7fe2ac901000 ---p 0000c000 fd:00 2228253                    /lib64/libnss_files-2.12.so
7fe2ac901000-7fe2ac902000 r--p 0000c000 fd:00 2228253                    /lib64/libnss_files-2.12.so
7fe2ac902000-7fe2ac903000 rw-p 0000d000 fd:00 2228253                    /lib64/libnss_files-2.12.so
7fe2ac903000-7fe2ac913000 r-xp 00000000 fd:00 135507                     /usr/lib64/python2.6/lib-dynload/datetime.so
7fe2ac913000-7fe2acb13000 ---p 00010000 fd:00 135507                     /usr/lib64/python2.6/lib-dynload/datetime.so
7fe2acb13000-7fe2acb17000 rw-p 00010000 fd:00 135507                     /usr/lib64/python2.6/lib-dynload/datetime.so
7fe2acb17000-7fe2acb1b000 r-xp 00000000 fd:00 135490                     /usr/lib64/python2.6/lib-dynload/_localemodule.so
7fe2acb1b000-7fe2acd1b000 ---p 00004000 fd:00 135490                     /usr/lib64/python2.6/lib-dynload/_localemodule.so
7fe2acd1b000-7fe2acd1c000 rw-p 00004000 fd:00 135490                     /usr/lib64/python2.6/lib-dynload/_localemodule.so
7fe2acd1c000-7fe2acd22000 r-xp 00000000 fd:00 2228451                    /lib64/libip6tc.so.0.0.0-1.4.7
7fe2acd22000-7fe2acf22000 ---p 00006000 fd:00 2228451                    /lib64/libip6tc.so.0.0.0-1.4.7
7fe2acf22000-7fe2acf23000 rw-p 00006000 fd:00 2228451                    /lib64/libip6tc.so.0.0.0-1.4.7
7fe2acf23000-7fe2acf29000 r-xp 00000000 fd:00 2228449                    /lib64/libip4tc.so.0.0.0-1.4.7
7fe2acf29000-7fe2ad128000 ---p 00006000 fd:00 2228449                    /lib64/libip4tc.so.0.0.0-1.4.7
7fe2ad128000-7fe2ad129000 rw-p 00005000 fd:00 2228449                    /lib64/libip4tc.so.0.0.0-1.4.7
7fe2ad129000-7fe2ad12a000 r-xp 00000000 fd:00 134101                     /usr/lib64/python2.6/site-packages/libxtwrapper.so
7fe2ad12a000-7fe2ad32a000 ---p 00001000 fd:00 134101                     /usr/lib64/python2.6/site-packages/libxtwrapper.so
7fe2ad32a000-7fe2ad32b000 rw-p 00001000 fd:00 134101                     /usr/lib64/python2.6/site-packages/libxtwrapper.so
7fe2ad32b000-7fe2ad332000 r-xp 00000000 fd:00 2228457                    /lib64/libxtables.so.4.0.0-1.4.7
7fe2ad332000-7fe2ad532000 ---p 00007000 fd:00 2228457                    /lib64/libxtables.so.4.0.0-1.4.7
7fe2ad532000-7fe2ad533000 rw-p 00007000 fd:00 2228457                    /lib64/libxtables.so.4.0.0-1.4.7
7fe2ad533000-7fe2ad5f4000 rw-p 00000000 00:00 0
7fe2ad5f4000-7fe2ad5f8000 r-xp 00000000 fd:00 135526                     /usr/lib64/python2.6/lib-dynload/selectmodule.so
7fe2ad5f8000-7fe2ad7f8000 ---p 00004000 fd:00 135526                     /usr/lib64/python2.6/lib-dynload/selectmodule.so
7fe2ad7f8000-7fe2ad7fa000 rw-p 00004000 fd:00 135526                     /usr/lib64/python2.6/lib-dynload/selectmodule.so
7fe2ad7fa000-7fe2ad7fc000 r-xp 00000000 fd:00 135510                     /usr/lib64/python2.6/lib-dynload/fcntlmodule.so
7fe2ad7fc000-7fe2ad9fc000 ---p 00002000 fd:00 135510                     /usr/lib64/python2.6/lib-dynload/fcntlmodule.so
7fe2ad9fc000-7fe2ad9fe000 rw-p 00002000 fd:00 135510                     /usr/lib64/python2.6/lib-dynload/fcntlmodule.so
7fe2ad9fe000-7fe2ada01000 r-xp 00000000 fd:00 135494                     /usr/lib64/python2.6/lib-dynload/_randommodule.so
7fe2ada01000-7fe2adc00000 ---p 00003000 fd:00 135494                     /usr/lib64/python2.6/lib-dynload/_randommodule.so
7fe2adc00000-7fe2adc01000 rw-p 00002000 fd:00 135494                     /usr/lib64/python2.6/lib-dynload/_randommodule.so
7fe2adc01000-7fe2adc05000 r-xp 00000000 fd:00 135502                     /usr/lib64/python2.6/lib-dynload/binascii.so
7fe2adc05000-7fe2ade05000 ---p 00004000 fd:00 135502                     /usr/lib64/python2.6/lib-dynload/binascii.so
7fe2ade05000-7fe2ade06000 rw-p 00004000 fd:00 135502                     /usr/lib64/python2.6/lib-dynload/binascii.so
7fe2ade06000-7fe2ade0d000 r-xp 00000000 fd:00 135498                     /usr/lib64/python2.6/lib-dynload/_struct.so
7fe2ade0d000-7fe2ae00d000 ---p 00007000 fd:00 135498                     /usr/lib64/python2.6/lib-dynload/_struct.so
7fe2ae00d000-7fe2ae00f000 rw-p 00007000 fd:00 135498                     /usr/lib64/python2.6/lib-dynload/_struct.so
7fe2ae00f000-7fe2ae016000 r-xp 00000000 fd:00 2361388                    /usr/lib64/libffi.so.5.0.6
7fe2ae016000-7fe2ae216000 ---p 00007000 fd:00 2361388                    /usr/lib64/libffi.so.5.0.6
7fe2ae216000-7fe2ae217000 rw-p 00007000 fd:00 2361388                    /usr/lib64/libffi.so.5.0.6
7fe2ae217000-7fe2ae231000 r-xp 00000000 fd:00 135480                     /usr/lib64/python2.6/lib-dynload/_ctypes.so
7fe2ae231000-7fe2ae430000 ---p 0001a000 fd:00 135480                     /usr/lib64/python2.6/lib-dynload/_ctypes.so
7fe2ae430000-7fe2ae435000 rw-p 00019000 fd:00 135480                     /usr/lib64/python2.6/lib-dynload/_ctypes.so
7fe2ae435000-7fe2ae436000 r-xp 00000000 fd:00 135499                     /usr/lib64/python2.6/lib-dynload/_weakref.so
7fe2ae436000-7fe2ae636000 ---p 00001000 fd:00 135499                     /usr/lib64/python2.6/lib-dynload/_weakref.so
7fe2ae636000-7fe2ae637000 rw-p 00001000 fd:00 135499                     /usr/lib64/python2.6/lib-dynload/_weakref.so
7fe2ae637000-7fe2ae63c000 r-xp 00000000 fd:00 135528                     /usr/lib64/python2.6/lib-dynload/stropmodule.so
7fe2ae63c000-7fe2ae83b000 ---p 00005000 fd:00 135528                     /usr/lib64/python2.6/lib-dynload/stropmodule.so
7fe2ae83b000-7fe2ae83d000 rw-p 00004000 fd:00 135528                     /usr/lib64/python2.6/lib-dynload/stropmodule.so
7fe2ae83d000-7fe2ae842000 r-xp 00000000 fd:00 135517                     /usr/lib64/python2.6/lib-dynload/mathmodule.so
7fe2ae842000-7fe2aea41000 ---p 00005000 fd:00 135517                     /usr/lib64/python2.6/lib-dynload/mathmodule.so
7fe2aea41000-7fe2aea43000 rw-p 00004000 fd:00 135517                     /usr/lib64/python2.6/lib-dynload/mathmodule.so
7fe2aea43000-7fe2aea46000 r-xp 00000000 fd:00 135489                     /usr/lib64/python2.6/lib-dynload/_json.so
7fe2aea46000-7fe2aec45000 ---p 00003000 fd:00 135489                     /usr/lib64/python2.6/lib-dynload/_json.so
7fe2aec45000-7fe2aec46000 rw-p 00002000 fd:00 135489                     /usr/lib64/python2.6/lib-dynload/_json.so
7fe2aec46000-7fe2aec4a000 r-xp 00000000 fd:00 135505                     /usr/lib64/python2.6/lib-dynload/cStringIO.so
7fe2aec4a000-7fe2aee49000 ---p 00004000 fd:00 135505                     /usr/lib64/python2.6/lib-dynload/cStringIO.so
7fe2aee49000-7fe2aee4b000 rw-p 00003000 fd:00 135505                     /usr/lib64/python2.6/lib-dynload/cStringIO.soAborted
```

**Comment by ldx** [comment]
Sorry about that! Let me set up a CentOS6 VM to check what is wrong.

Do you use latest master or 0.9.0?

**Comment by RickDavis84** [comment]
No problem, I'm going to upgrade my box to CentOS 7 so I can move forward...I was using the version in the commit you specified - 942a50d - which was 0.9.0-dev

**Comment by ldx** [comment]
In the meantime I released 0.9.0, can you check that version too?

**Comment by RickDavis84** [comment]
Sure thing, i'll give it a shot this evening
